rust/gui-client/src-tauri/src/controller.rs

```diff
@@ -550,21 +565,21 @@ impl<I: GuiIntegration> Controller<I> {
                     "Uptime info"
                 );
             }
-            Req::SystemTrayMenu(system_tray::Event::SignOut) => {
+            SignOut | SystemTrayMenu(system_tray::Event::SignOut) => {
```

Comment:
Clicking `SignOut` just triggers the same codepath as clicking it in the tray menu.